src/riak_cs_config.erl

```diff
@@ -488,6 +462,7 @@ local_get_block_timeout() ->
 ?TIMEOUT_CONFIG_FUNC(get_index_range_gckeys_timeout).
 ?TIMEOUT_CONFIG_FUNC(get_index_range_gckeys_call_timeout).
 ?TIMEOUT_CONFIG_FUNC(get_index_list_multipart_uploads_timeout).
+?TIMEOUT_CONFIG_FUNC(cluster_id_timeout).
```

Comment:
From the implicit naming convention on these timeouts, this would better be `get_clusterid_timeout`.